library/keystone_role.py

```diff
@@ -19,9 +19,11 @@
 
 
 try:
+    from keystoneauth1 import session
```

Comment:
Is there a reason we're still using this module, instead of the `os_keystone_role` module we use elsewhere?